.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, macos-12, ubuntu-20.04]
+        os: [windows-2022, ubuntu-20.04, macos-12, macos-13-xlarge]
```

Comment:
this one can be kept

Reply:
did you make sure that the binaries were not overriding each others once uploaded?

Reply:
Confirmed the binaries are correct for each platform.